underscore.js

```diff
@@ -1489,11 +1508,10 @@
   _.mixin = function(obj) {
     _.each(_.functions(obj), function(name) {
       var func = _[name] = obj[name];
-      _.prototype[name] = function() {
-        var args = [this._wrapped];
-        push.apply(args, arguments);
+      _.prototype[name] = restArgs(function(args) {
```

Comment:
It actually looks like it's [slower](http://jsperf.com/restargs-mixin). @megawac?